src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -579,4 +595,46 @@ p5.RendererGL.prototype._drawImmediateStroke = function() {
   shader.unbindShader();
 };
 
+/**
+ * Called from image rendering functions. Responsible for setting shader uniforms,
+ * enabling all appropriate buffers, binding textures, applying color blend,
+ * and drawing the image geometry.
+ * @private
+ */
+p5.RendererGL.prototype._drawImmediateImage = function(count = 1) {
```

Comment:
Could probably remove the whole function. Since the logic with merging it, `this._drawingImage` will take care of that.